src/spacel/model/manifest.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, params={}):
                         for label, vol_params in
                         params.get('volumes', {}).items()}
         self.elb = params.get('elb')
+        self.elb_time = params.get('elb_time', 30)
```

Comment:
+1; just took another look at https://github.com/pebble/spacel-agent/blob/master/src/spacel/agent/instance.py this API should be more like it for consistency.

Will refactor+fold.